From: Zhang Lixu <lixu.zhang@intel.com>
Date: Thu, 17 Apr 2025 11:13:05 +0800
Subject: [PATCH] intel: ish: Update license file for ISH
Origin: https://gitlab.com/lixuzha/linux-firmware/-/commit/57d650b7f8eb986f2b117f8fe4a603363fb9a2ad
Bug-Debian: https://bugs.debian.org/1102999

Update the license file for ISH to resolve compliance issue

Signed-off-by: Zhang Lixu <lixu.zhang@intel.com>

[bwh: Delete the change to LICENCE.ish, which we exclude from the upstream source]
---
 LICENCE.ish   | 173 --------------------------------------------------
 LICENSE.intel |  35 ++++++++++
 WHENCE        |   2 +-
 3 files changed, 36 insertions(+), 174 deletions(-)
 delete mode 100644 LICENCE.ish
 create mode 100755 LICENSE.intel

diff --git a/LICENSE.intel b/LICENSE.intel
new file mode 100755
index 00000000..c4b92257
--- /dev/null
+++ b/LICENSE.intel
@@ -0,0 +1,35 @@
+Copyright (c) 1999-2025 Intel Corporation
+
+Redistribution.  Redistribution and use in binary form, without
+modification, are permitted provided that the following conditions are
+met:
+
+* Redistributions must reproduce the above copyright notice and the
+  following disclaimer in the documentation and/or other materials
+  provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its suppliers
+  may be used to endorse or promote products derived from this software
+  without specific prior written permission.
+* No reverse engineering, decompilation, or disassembly of this software
+  is permitted.
+
+Limited patent license.  Intel Corporation grants a world-wide,
+royalty-free, non-exclusive license under patents it now or hereafter
+owns or controls to make, have made, use, import, offer to sell and
+sell ("Utilize") this software, but solely to the extent that any
+such patent is necessary to Utilize the software alone. The patent 
+license shall not apply to any other combinations which include this 
+software.  No hardware per se is licensed hereunder.
+
+DISCLAIMER.  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+DAMAGE.
diff --git a/WHENCE b/WHENCE
index d987f2b7..f2dcf32c 100644
--- a/WHENCE
+++ b/WHENCE
@@ -8471,7 +8471,7 @@ Driver: ish - Intel Integrated Sensor Hub
 File: intel/ish/ish_lnlm.bin
 Version: 5.8.0.7716
 
-License: Redistributable. See LICENCE.ish for details
+License: Redistributable. See LICENSE.intel for details
 
 --------------------------------------------------------------------------
 
